app/src/main/java/com/framgia/wsm/screen/main/MainViewModel.java

```diff
@@ -224,7 +227,8 @@ private String titleWorkingCalendar() {
         int HOLIDAY_CALENDAR = 3;
         int STATISTIC_OF_PERSONAL = 4;
         int OVERTIME = 5;
-        int COME_LATE_LEAVE_EARLY = 6;
-        int WORKSPACE_DATA = 7;
+        int OFF = 6;
+        int COME_LATE_LEAVE_EARLY = 7;
```

Comment:
ERROR: 'member def type' have incorrect indentation level 8, expected level should be 4.